secrets/gcp.go

```diff
@@ -112,10 +123,117 @@ func (g *gcpSecretsManager) CheckSecretExists(ctx context.Context, key string) b
 	return err == nil
 }
 
+func (g *gcpSecretsManager) CreateSecret(ctx context.Context, key string, value string, labels map[string]string) error {
+	secretID := g.getSecretID(key)
+	mergedLabels := g.mergeTags(labels)
+
+	// Create the secret
+	_, err := g.client.CreateSecret(ctx, &secretmanagerpb.CreateSecretRequest{
+		Parent:   fmt.Sprintf("projects/%s", g.projectID),
```

Comment:
I wonder as per docs:
> 	// Required. The resource name of the project to associate with the
	// [Secret][google.cloud.secretmanager.v1.Secret], in the format `projects/*`
	// or `projects/*/locations/*`.

Should you include locations?